### PR TITLE
Cody Windeknecht --- Sprint-Challenge--SQL

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by cwindeknecht** [comment]
Not too shabby since I didn't even start until 1pm... didn't know new PMs were supposed to be doing this until Alex Figliolia mentioned it around then.

File: queries.sql [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    channel_id INTEGER REFERENCES channel(id),
+    user_id INTEGER REFERENCES user(id),
+    dt DATETIME DEFAULT current_timestamp,
+    content VARCHAR(128) NOT NULL
+);
+
+INSERT INTO organization (org_name) VALUES ("Lambda School");
+
+INSERT INTO user (username,organization_id) VALUES ("Alice",1);
+INSERT INTO user (username,organization_id) VALUES ("Bob",1);
```
Comment (Velsu) [inline queries.sql:41]: Just a tip that You can also write multiple `INSERT` queries in one line by separating values with coma.

File: queries.sql [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+-- 9. List the count of messages across all channels per user.
+SELECT user.username,COUNT(*) FROM user,channel_message
+    WHERE channel_message.user_id = user.id
+    GROUP BY user.id
+    ORDER BY user.username DESC;
+
+-- 10. [Stretch!] List the count of messages per user per channel.
+SELECT user.username, channel.ch_name,
+    SUM(CASE WHEN channel_message.user_id = user.id THEN 1 ELSE 0 END)
```
Comment (Velsu) [inline queries.sql:123]: This is really nice.

**Review by Velsu — COMMENTED** [review]
Amazing job on the sprint challenge. Great way to write queries and You even completed stretch goal.

**Pull request closed without merging**